src/main/java/at/hannibal2/skyhanni/config/features/event/carnival/ZombieShootoutConfig.java

```diff
@@ -21,14 +21,19 @@ public class ZombieShootoutConfig {
     public boolean coloredHitboxes = true;
 
     @Expose
-    @ConfigOption(name = "Highest Only", desc = "Only draw colored hitboxes for the highest scoring zombies.")
+    @ConfigOption(name = "Zombie Timer", desc = "Displays a timer above the heads of the zombies.")
+    @ConfigEditorBoolean
+    public boolean zombieTimer = true;
+
+    @Expose
+    @ConfigOption(name = "Highest Only", desc = "Only draw colored hitboxes/timers for the highest scoring zombies.")
     @ConfigEditorBoolean
     public boolean highestOnly = false;
 
     @Expose
     @ConfigOption(name = "Colored Line", desc = "Display a colored line to lamps.")
     @ConfigEditorBoolean
-    public boolean coloredLines = true;
```

Comment:
I dont really see the point in changing this, but if its changed it needs a config fix

Reply:
I did this before it got merged initially, just wanted the val name to be the same as the name, so line instead of lines